Books/client/src/features/misc/routes/SearchPage/SearchPage.scss

```diff
@@ -0,0 +1,4 @@
+.searchPageTitle {
+  padding: 40px;
+  text-align: center;
```

Comment:
There is a `text-center` utility class which center aligns text and I think that would be better suited to this job.